### PR TITLE
point to new email and fix FT test

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

Enter a description.
pointing to copy of original template plus new text of email.
Plus fixing the test to use uk address and add test for overseas one.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-2747

**Before merging a pull request make sure that:**

- [X] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/functionalTest/resources/responses/response-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json [hunk 1]
```diff
@@ -0,0 +1,142 @@
+{
+  "data": {
```
Comment (benouaer) [inline src/functionalTest/resources/responses/response-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json:2]: is all this data needed for this specific callback? might be easier to manage if you just restrict it to the data used in the callback

File: src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
```
Comment (benouaer) [inline src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json:1]: is all this data needed for this specific callback? might be easier to manage if you just restrict it to the data used in the callback

File: src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-about-to-submit.json [hunk 3]
```diff
@@ -13,10 +13,11 @@
     "Country": "United Kingdom"
   },
   "applicant2Address": {
-    "AddressLine1": "Parc de la Ciutadella",
```
Comment (benouaer) [inline src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-about-to-submit.json:16]: is all this data needed for this specific callback? might be easier to manage if you just restrict it to the data used in the callback

File: src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json (hunk 2 above)
Comment (velimd) [reply src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json:1]: im not sure as I copied it from a similar test with some changes to the request I made.

File: src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-about-to-submit.json (hunk 3 above)
Comment (velimd) [reply src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-about-to-submit.json:16]: im not sure as I just updated the address to use the UK one and created a similar test with overseas address.

**Pull request merged**